reviewer/metrics/Metrics.hh

```diff
@@ -20,16 +20,18 @@
 
 #pragma once
 
+#include <ostream>
```

Comment:
This trend of moving {io,i,o,s}stream headers from the cpp to the headers is generally the opposite of what you want for good compilation speed -- typically a header will just have <iosfwd> if iostream is part of the interface.

Reply:
Good to know! Done.